src/presentation/controllers/device-controller.ts

```diff
@@ -0,0 +1,131 @@
+import { BaseController, DeleteRequest, GetRequest, PostRequest, PutRequest, RestController } from './base-controller'
+import { Request, Response } from 'express'
+import { DeviceService } from '../../business-logic/services/interfaces/device-service'
+import { HttpErrorHandler } from '../interfaces/http-error-handler'
+import { Exception } from '../../model/exceptions/exception'
+import { HttpResponseFormatter } from '../interfaces/http-response-formatter'
+import { DeviceDto } from '../dtos/device-dto'
+import { HealthDto } from '../dtos/health-dto'
+import { DeviceRestDto } from '../../model/entities/device'
+import { INewsDevice, INewsDeviceRestDTO } from '../../model/entities/inews-device'
+import { TelemetricsDevice, TelemetricsDeviceRestDTO } from '../../model/entities/telemetrics-device'
+
+@RestController('/devices')
+export class DeviceController extends BaseController{
+
+  constructor(
+    private readonly deviceService: DeviceService,
+    private readonly httpErrorHandler: HttpErrorHandler,
+    private readonly httpResponseFormatter: HttpResponseFormatter
+  ) {
+    super()
+  }
+
+  @GetRequest('/health')
+  public health(_request: Request, response: Response): void {
+    try {
+      const healthcheck = {
+        statusCode: 200,
+        state: 'Healthy'
+      }
+      response.send(this.httpResponseFormatter.formatSuccessResponse(new HealthDto(healthcheck.statusCode, healthcheck.state)))
+    } catch (error) {
+      this.httpErrorHandler.handleError(response, error as Exception)
+    }
+  }
+
+  @GetRequest('/')
```

Comment:
I dont know what the norm is, but you don't need the / here :) 
```suggestion
  @GetRequest()
```